### ISSUE TITLE
发版前需要检查的内容

### ISSUE THREAD

**Issue by Ldpe2G** [Oneflow-Inc/models#149]
## 需要检查的事项

1.  oneflow 用最新的 master 分支 836fa34500c54610f20ddaf9e537c0878a775062
2. 确认模型的推理脚本能正常运行，结果与 readme 中的描述一致
3. 训练脚本也要跑 ，看下收敛情况就行
4. 

## 记录表

MODEL | 负责人 |  描述完成情况，检查了哪些内容，遇到什么问题
-- | -- | -- 
CSRNet |   @rentainhe | 已完成
DCGAN | @MARD1NO  |  修复了reshape传参，脚本路径，权重路径的问题
DLA | @CPFLAME   |   已完成，删除visdom相关的代码和readme，https://github.com/Oneflow-Inc/models/pull/172
FODDet |@zhangshen12356  |   1、按照readme中需要自己下载数据集 2、train.sh和test.sh能够正常运行
FaceSeg | @rentainhe   |   之江小伙伴帮忙修改好了，已完成
LSTMText | @xiezipeng-ML  |   解决了一下文件路径和注释问题，可以正常训练 https://github.com/Oneflow-Inc/models/pull/168
SRGAN | @MARD1NO  |  其中有一行漏写了个 mean，存值的时候只取了第一个元素。另外就是0dshape的问题，已把多余的[0]都去除
SincNet  |  @lcylcy  |   已完成，解决flow.f.conv1d找不到，移植到flow.functional.conv1d
TSN   |  @zhangshen12356 |   在运行test.sh时会报错“input shape of parameter of reshape is not illegal"，已修复https://github.com/Oneflow-Inc/models/pull/183
TextCNN   | @xiezipeng-ML  |   完成
Transformer   |  @L1aoXingyu |   已修复 https://github.com/Oneflow-Inc/models/pull/156
Wav2Letter   |  @lcylcy  |   已完成，解决数据读取问题。
alexnet   | @rentainhe   |   已完成
bert-oneflow   | @L1aoXingyu   |   已完成 
cycleGAN   |@CPFLAME   |   1. 下载数据集时，需要先输入指令`mkdir datasets`，在readme中没有写。 2. 运行train.sh失败，函数接口有变动`flow.optim.lr_scheduler.CosineAnnealingLR(optimizer, steps=100, alpha=0.0) got an unexpected keyword argument 'steps'`  <br> fix pr: https://github.com/Oneflow-Inc/models/pull/154
densenet    | @rentainhe  |   已完成
fast_neural_style   |  @CPFLAME |  fix pr:  https://github.com/Oneflow-Inc/models/pull/157
ghostnet   | @MARD1NO  |  预训练权重路径不对，已修改
inception_v3 | @BBuf  | 已完成
mobilenetv2 | @rentainhe  | 已完成
mobilenetv3 | @BBuf | 已完成
pix2pix | @CPFLAME  |  fix pr: https://github.com/Oneflow-Inc/models/pull/158
poseNet | @CPFLAME  |
quantization | @BBuf  | 已完成
quick_start_demo_lenet | @rentainhe | 已完成
reid | @L1aoXingyu  | 已修复 https://github.com/Oneflow-Inc/models/pull/152
repvgg | @MARD1NO  | 过时的initEagerxxx，已去除
resnext50_32x4d| @rentainhe  | 已完成
rnn | @xiezipeng-ML | 完成
scnet | @rentainhe  | 已完成
seq2seq  |  @xiezipeng-ML  | 完成
shufflenetv2   | @BBuf  | 已完成
vgg | @MARD1NO  | 暂无问题
speaker_identification_demo | @lcylcy  |已完成
